test/vcr_clj/test/cassettes/serialization.clj

```diff
@@ -0,0 +1,10 @@
+(ns vcr-clj.test.cassettes.serialization
+  (:require [clojure.test :refer :all]
+            [vcr-clj.cassettes.serialization :refer :all])
+  (:import [java.io InputStream]))
+
+(deftest can-read-input-stream
+  (testing "simple base64 encoded string"
+    (is (instance? InputStream (read-input-stream "aGVsbG8="))))
+  (testing "empty string"
+    (is (instance? InputStream (read-input-stream "")))))
```

Comment:
without much effort you could take these tests a bit farther by calling `slurp` on the inputstream and checking the decoded result.